lib/features/search-pad/SearchPad.js

```diff
@@ -449,6 +453,21 @@ SearchPad.prototype._resetOverlay = function(element) {
   }
 };
 
+SearchPad.prototype._getBoxHtml = function() {
+  const box = domify(
+    '<div class="djs-search-container djs-draggable djs-scrollable">' +
+      '<div class="djs-search-input">' +
+        '<input type="text"/>' +
+      '</div>' +
+      '<div class="djs-search-results"></div>' +
+    '</div>'
+  );
+
+  box.children[0].children[0].setAttribute('aria-label', this._translate('Search'));
```

Comment:
Let's use `SearchPad.BOX_HTML` as a foundation and try to find the input by selector. We'll otherwise risk to break user code unnecessarily.

Reply:
Q: Shall the label provide more context, i.e. "search diagram"?

Reply:
ok

Reply:
OK done